### ISSUE TITLE
Add more Comparison Operators

### ISSUE THREAD

**Issue by UmanGarbag** [claranet/nagitheus#18]
Hello, 

I using you tool to query some Prometheus Instance and create alert on icinga2 based on the result, i figure out that i need to check if a value is equal or not equal.

So i made a little modification to our code, by adding more function for this usage.

```
func (c *Comparison) EQ() bool {
        return c.x == c.y
}
```
```
func (c *Comparison) NQ() bool {
        return c.x != c.y
}
```

**Comment by teresabroenner** [comment]
Hi @UmanGarbag 

Thank you for your suggestion! We added it in version 1.8.0 of Nagitheus.
Therefore the issue will be closed

**Comment by teresabroenner** [comment]
Released with 1.8.0
